scripts/database-exporter/Dockerfile

```diff
@@ -1,25 +0,0 @@
-ARG base_image_tag
```

Comment:
These files seems to have been forgotten to be removed after some duplication occurred.